sucuri.php

```diff
@@ -8,7 +8,7 @@
  * Author: Sucuri Inc.
  * Text Domain: sucuri-scanner
  * Domain Path: /lang
- * Version: 1.8.36
+ * Version: 1.8.37
```

Comment:
Could you also update the `define('SUCURISCAN_VERSION', '1.8.36');` in line 88?

Reply:
Done! Thank you very much!